spec.html

```diff
@@ -30549,7 +30549,7 @@ <h1>CharacterClassEscape</h1>
             1. Return the CharSet containing all Unicode code points whose character database definition includes the property &ldquo;General_Category&rdquo; with value _LoneUnicodePropertyNameOrValue_.
           1. Let _p_ be ! UnicodeMatchProperty(_LoneUnicodePropertyNameOrValue_).
           1. Assert: _p_ is a binary Unicode property or binary property alias listed in the &ldquo;Property name and aliases&rdquo; column of <emu-xref href="#table-binary-unicode-properties"></emu-xref>.
-          1. Return the CharSet containing all Unicode code points whose character database definition includes the property _p_ with value |True|.
+          1. Return the CharSet containing all Unicode code points whose character database definition includes the property _p_ with value &ldquo;True&rdquo;.
```

Comment:
Should the value `true` be lowercased?

Reply:
The current casing is correct. The binary property values according to the Unicode Standard are `N`/`No`/`F`/`False` and `Y`/`Yes`/`T`/`True`.

Reply:
gotcha, I read it as "JS value" :-) thanks for clarifying.